sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -0,0 +1,483 @@
+# Azure Form Recognizer client library for JavaScript
```

Comment:
now that Python has updated the [Readme ](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/formrecognizer/azure-ai-formrecognizer#readme)according to Paul's feedback, I suggest you look at that and update this file

Reply:
Done, thanks!